### PR TITLE
[FWF-2483] Component changes for ELK support

### PR DESCRIPTION
**Pull request by SnehaSuresh-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE
https://aottech.atlassian.net/browse/FWF-2483

# Changes
* added error log file + cleanup

# How has the change been tested?

# Screenshots (if applicable)
image
image

# Build Success screenshot (Till a CICD pipeline is set up)
image

# Notes

### CODE REVIEW

File: forms-flow-bpm/src/main/resources/application.yaml [hunk 1]
```diff
@@ -182,4 +182,10 @@ logging:
...
     org.springframework.web: ${CAMUNDA_APP_ROOT_LOG_FLAG}
     org.springframework.jdbc: ${CAMUNDA_APP_ROOT_LOG_FLAG}
-    org.camunda.bpm: ${CAMUNDA_APP_ROOT_LOG_FLAG}
\ No newline at end of file
+    org.camunda.bpm: ${CAMUNDA_APP_ROOT_LOG_FLAG}
+  pattern:
+    console: '%d{yyyy-MM-dd HH:mm:ss} - %msg%n'
+    file: '%d{yyyy-MM-dd HH:mm:ss} [%thread] %-5level %logger{36} - %msg%n'
+  file:
+    name: error.log
```
Comment (sumesh-aot) [inline forms-flow-bpm/src/main/resources/application.yaml:190]: may be change the name to just bpmlogs or something generic as logs would contain any log level based on configuration ?

**Review by sumesh-aot — APPROVED** [review]

File: forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/FormBPMDataPipelineListener.java [hunk 2]
```diff
@@ -20,6 +22,8 @@
 @Named("FormBPMDataPipelineListener")
```
Comment (shibin-aot) [inline forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/FormBPMDataPipelineListener.java:22]: Moving out from Java -DI to Spring DI can be considered.

File: forms-flow-bpm/src/main/resources/application.yaml (hunk 1 above)
Comment (SnehaSuresh-aot) [reply forms-flow-bpm/src/main/resources/application.yaml]: yes done.

File: forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/FormBPMDataPipelineListener.java (hunk 2 above)
Comment (SnehaSuresh-aot) [reply forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/FormBPMDataPipelineListener.java:22]: Yes, Of course. `@Named` support has been added in Spring `3.0,` we are now using `2.7.12` version.

**Review by auslin-aot — APPROVED** [review]

**Review by abhilash-aot — APPROVED** [review]

**Review by abhilash-aot — APPROVED** [review]

**Pull request merged**